ChessCrafterDesktop/src/main/resources/GuiStrings_en_US.properties

```diff
@@ -130,6 +130,12 @@ Driver.startNewGame=Start a new Game
 Driver.sureYouWannaQuit=Are you sure you want to Quit?
 Driver.variants=Variants
 Driver.viewCompleted=View Completed Game
+GamePanel.inCheck=You are in Check!
+GamePanel.info=Player Info
+GamePanel.myTurn=Your turn

```

Comment:
Same thing about "Your" here, but i'm not sure what a better phrase is. Maybe "Moving"?